src/components/Router/StepComponentMap.js

```diff
@@ -44,7 +44,7 @@ const createDocumentComponents = (documentType) => {
 
 const crossDeviceSteps = (steps) => {
   const baseSteps = [{'type': 'crossDevice'}]
-  const completeStep = steps.find(isComplete)
+  const completeStep = Array.find(steps, isComplete)
```

Comment:
Running in BrowserStack it looks like `Array.find` also isn't supported (which makes sense)

Are you sure this works?

Reply:
Ahh, I was checking on the global scope. So I assume you get it from some polyfill that just doesn't correctly polyfill the actual array prototype to allow you do to `[].find` on IE11

Does that sound right?

Reply:
Yes, we get if from `core-js`.